### ISSUE TITLE
How can i change property of "nestedProps" depending on conditional

### ISSUE THREAD

**Issue by jongjunpark** [figma/code-connect#118]
- **`stickyHeader: false`** -> has `type`, `withDescription`, `closeButton` properties
<img width="329" alt="image" src="https://github.com/user-attachments/assets/73220e05-cd9e-4485-8f06-7b3003b0e2ff">

- **`stickyHeader: true`** -> hide `type`, `withDescription`, `closeButton` properties
<img width="351" alt="image" src="https://github.com/user-attachments/assets/f27a0e17-fe44-4d9d-8a83-b0018c12837d">

How can I control it?

**Comment by tomduncalf-figma** [comment]
Hi, could you share your Code Connect for this? I'm not 100% clear on what the question is. Thanks

**Comment by jongjunpark** [comment]
Hi, @tomduncalf-figma.
If `stickyHeader` is set to false, then nestedProps("Desktop") should have the properties `type`, `withDescription`, and `closeButton`. On the other hand, if `stickyHeader` is set to true, then nestedProps("Desktop") should not include these properties.

I think it can be set up like this code, 
```javascript
      stickyHeader: figma.boolean("stickyHeader", {
        true: undefined,
        false: figma.nestedProps("_modalHeader/Desktop", {
          type: figma.enum("type", {
            leftAlign: "leftAlign",
            centerAlign: "centerAlign",
          }),
          closeButton: figma.boolean("closeButtton"),
          showHeadingText: figma.boolean("showHeadingText"),
        }),
      }),
```
but I'm getting an error saying "Could not find prop mapping for stickyHeader in the props object"

**Comment by jongjunpark** [comment]
Additionally, even though I set stickyHeader.type as an enum, it shows up as a string instead of an enum.

**Comment by tomduncalf-figma** [comment]
It might be easiest if we can see your Figma file and your full Code Connect file for this case. Assuming you don't want to share that publicly, you can contact [Figma support](https://help.figma.com/hc/en-us/requests/new), mentioning that you have been referred from Github and would like to share your files with the team, and they have a process to give us temporary access

**Comment by karlpetersson** [comment]
Hey! Just wanted to follow up here that we're including a fix for using `figma.nestedProps` in `figma.boolean` in the next release - it looks like it will hopefully solve this issue for you, will follow up here once we've made the release

**Comment by tomduncalf-figma** [comment]
We just released Code Connect v1.1.1 which should resolve this issue. Please feel free to reopen the issue or create a new one if you continue to have issues.
